documentation/learnmore/getting-certified.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: documentation
+menu: gettingCertified
+subtitle: Getting your Database Certified
+---
+
+# Getting Flyway Certified support for your database
+
+This page provides guidance on the process and crtieria for getting your database added to Flyway’s list of “Certified supported” databases. It’s aimed at the vendors and maintainers of relational database management systems (“RDBMS”). 
+
+_If you’re an end user of a database Flyway doesn’t support, and you would like it to, you should let us know by raising an issue on our GitHub repo, or by following the how-to guide for [making your database compatible with Flyway](/documentation/contribute/contributingDatabaseSupport) yourself._
+
+There are three levels of database support; ordered lowest to highest these are **Compatible**, **Certified** and **Guaranteed**. Each reflects the level of confidence we have in our ability to service Flyway users when using those RDBMSs. 
+
+You can read more about [Flyway Database Support Levels](/documentation/learnmore/database-support) but in short, for a database to achieve Certified (or higher) support status, we require it to meet certain criteria. 
+
+## Background
+
+At first glance, adding support for a new engine seems trivial and a lot of the time, mechanically, it is. In fact, when support for an engine isn't available out of the box, we provide end users with [documentation to make Flyway compatible](/documentation/contribute/contributingDatabaseSupport) with their database of choice.     
+
+On principle, we try to make sure any database we list as Certified is available in every edition of Flyway, including our free Community Edition. We want Flyway to be robust and reliable in any application, but stability is especially important for customers of our paid-for editions. These editions help keep the lights on and the Community Edition free. 
+
+One of the things that sets Flyway apart is its extensive and comprehensive approach to testing. Today we have well over 3000 individual tests. These tests ensure that Flyway is dependable in the most demanding infrastructure environments.
+
+Each new database deserves the same degree of testing. As you can imagine, this adds some overhead to Flyway's core development team.
+
+Our relationships with database vendors is an important part of being a stable product, and it's a relationship that also benefits vendors. Today, the majority of databases that have grown in popularity operate on some form of the open-core model, providing paid-for editions of their technology in much the same way as Flyway.
+
+There are [339 database engines](https://db-engines.com/en/ranking) in the world, offering a lot of choice for developers, their specific needs and context of application. When we add support into Flyway for a new database, we strengthen the ecosystems of those vendors and their databases.
```

Comment:
There are almost certainly way more than 339, but certainly 339 _serious_ ones...

Reply:
simplified in latest commit